### PR TITLE
Added data_presentation.ipynb with combined plot 

### PR DESCRIPTION
**Pull request by dhamalakamal**
For this assignment all pull requests going  forward should contain the following (please delete the placeholder text and add your own description)

# Title
  - Combined plot of co2 raw data, fft and inverse fft data
   
## Linting
  - Yes, all code follow lint structure.

## Contents
  - Added combined plot for raw data of co2 in timeseries and fft vs inverse fft data
  - new file with `data_presentation.ipynb` is added
  - https://github.com/ubsuny/CP1-24-HW5#task-group-3-data-presentation-max-6-members -> first task

## Labels and Milestones
  - ALL pull requests must have an attached label and Milestone before a reviewer can be assigned.

## Reviewer
  - @laserlab @HeshamElsaman 

Any further questions on formatting pull requests can be asked in the issues tab or on discord. 
This document can be changed at any time.

### CODE REVIEW

File: data_presentation.ipynb [hunk 1]
```diff
@@ -0,0 +1,132 @@
+{
```
Comment (laserlab) [inline data_presentation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #53.        # Subplot 2: FFT Magnitude vs Frequency
</pre><p>This is not needed, just plot the time docmain</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1-24-HW5/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by laserlab — CHANGES_REQUESTED** [review]
- plots have to work
- any data processing needed
- no fft

File: data_presentation.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhamalakamal) [reply data_presentation.ipynb:1]: No, actually the methods from preparation.py module are not returing data back instead I am getting None value which is why my further code for plotting isn't working. If the method fft_mag returns data correctly, it is expected to work. I can't think of a fix for that unless I understand why it's not returning the fft result back.

[the remark this answers is not in the document]
Comment (dhamalakamal) [reply data_presentation.ipynb:1]: I am getting it like this 
image

**Review by laserlab — APPROVED** [review]

**Pull request merged**